apcd-cms/src/apps/admin_submissions/views.py

```diff
@@ -63,6 +63,8 @@ def filtered_submissions(self, submission_content, status, sort):
         def getDate(submission):
             date = submission['received_timestamp']
             return parser.parse(date) if date is not None else parser.parse('1-1-3005')
+        
+        ## Initial API response sort
         submission_list = sorted(
```

Comment:
This sort initially is redundant. The sorting on line 76 will take care of the date sorting.

Reply:
Good catch! Thank you